payments-core/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -180,5 +173,22 @@ internal abstract class BaseAddPaymentMethodFragment(
                 SupportedPaymentMethod.Card -> CardDataCollectionFragment::class.java
                 else -> ComposeFormDataCollectionFragment::class.java
             }
+
+        private val transformToPaymentMethodCreateParams = TransformToPaymentMethodCreateParams()
+        private fun transformToPaymentSelection(
```

Comment:
any reasons why these are defined in companion object?

Reply:
I think it just makes it clear that it isn't using any (non-static) member variables.